examples/src/lib.rs

```diff
@@ -3,8 +3,11 @@
 // This source code is licensed under the MIT license found in the
 // LICENSE file in the root directory of this source tree.
 
+use fibonacci::{Blake3_192, Blake3_256, Sha3_256};
```

Comment:
It feels a bit weird that we import these here from one of the examples. Should we define them here and import them in examples instead? Or does this create issues?

Reply:
Agreed, makes sense. I've moved it to the the `lib.rs`. Please see [fa8e7c4](https://github.com/novifinancial/winterfell/pull/111/commits/fa8e7c408cd9ab80ae3367d8c68b87c9e2231ce1).